windows/windows_update_install/prepare_msu_file.yml

```diff
@@ -22,8 +22,15 @@
     msu_file_path: ""
     msu_nfs_path_list: []
 
+# For OS older than 17763 (Windows 1809 or Windows Server 2019), it has very high frequency of timeout for the command.
```

Comment:
I think this comment can be added for below set_fact task, as it is explanation about why use a fixed driver letter for OS older than 17763

Reply:
Moved. Thanks.